rom/dev/src/main.rs

```diff
@@ -89,6 +94,10 @@ pub extern "C" fn rom_entry() -> ! {
         Err(err) => report_error(err.into()),
     }
 
+    // Stop the watchdog timer.
+    // [TODO] Reset the watchdog timer and let FMC take ownership of it.
```

Comment:
FWIW, I think you should keep the watchdog timer going; the FMC can extend the delay if it wants to, but generally I would expect us to get all the way to runtime, where it turns off the watchdog right before it goes to sleep waiting for mailbox commands.

Reply:
That was the idea; I'll file an issue on runtime to add the WDT functionality and will remove the stop operation in the ROM.